engines/osxnative/sandboxbuilder.go

```diff
@@ -0,0 +1,33 @@
+package osxnative
+
+import (
+	"github.com/taskcluster/taskcluster-worker/engines"
+	"github.com/taskcluster/taskcluster-worker/runtime"
+)
+
+type sandboxbuilder struct {
+	engines.SandboxBuilderBase
+	env         map[string]string
+	taskPayload *payload
+	context     *runtime.TaskContext
+}
+
+func (s sandboxbuilder) SetEnvironmentVariable(name string, value string) error {
+	_, ok := s.env[name]
+	if ok {
+		return engines.ErrNamingConflict
+	}
+
+	s.context.Log("Setting \"", name, "\" environment variable to \"", value, "\" ")
```

Comment:
Hrm, could you point me to where they're needed for automated tests? I wonder if there is a way we could get around that being needed